Calibration/EcalCalibAlgos/test/EcalPhiSymReco_cfg.py

```diff
@@ -0,0 +1,72 @@
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+# Cmd line options
+options = VarParsing ('analysis')
+options.register('produceByRun',
+                 True,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Accumulate EcalPhiSym RecHits by Run or by LuminosityBlock")
+options.register('saveFlatNano',
+                 True,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce FlatNanoAOD output")
+options.register('saveEDMNano',
+                 False,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce EDMNanoAOD output")
+options.register('saveEDM',
+                 False,
+                 VarParsing.multiplicity.singleton,
+                 VarParsing.varType.bool,
+                 "Produce stadard EDM output")
+options.inputFiles=[""]
+options.parseArguments()
+
+process = cms.Process("ECALPHISYM")
+process.load("FWCore.MessageService.MessageLogger_cfi")
+
+# Multi-threading
+process.options.numberOfStreams = cms.untracked.uint32(4)
+process.options.numberOfThreads = cms.untracked.uint32(4)
+
+# Conditions
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('Configuration.Geometry.GeometryExtended2018Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2018Reco_cff')
```

Comment:
Is it intended to use 2018 configurations?

Reply:
just for testing now, we will have to discuss the actual workflow for 2022 more in details

Reply:
ok then!